.changeset/tidy-news-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+"api": patch
+---
+
+[sdk] move sdk
```

Comment:
Should be empty I think (we don't publish the `api` directory).

```suggestion
---

```

Reply:
Yup. I added `api` to changesets ignore: https://github.com/vercel/vercel/pull/12700/commits/7e862f7f73a4dff5ec3fd265f4a8dcb3dee30948